### ISSUE TITLE
Some problems with the accuracy of reproduction

### ISSUE THREAD

**Issue by fengpany** [vaibhavsaxena11/cwvae#6]
I reproduced the tensorflow version of the CWVAE code, but the accuracy of the reproduction is very different from the paper. Specifically, when testing the minerl_nvigate dataset (where num_examples=225, num_samples=1), the PSNR and SSIM are 0.55 and 17.3, respectively (which I averaged all the frames predicted in the minerl_nvigate test set), but this is much different from the results of the paper of 0.65 and 23.23. Where is the problem?How can I get thesis accuracy？
![image](https://github.com/vaibhavsaxena11/cwvae/assets/82636502/db4380a0-f2bd-4cae-be05-b1a32c45f1be)

**Comment by vaibhavsaxena11** [comment]
Hey, thank you for your interest in this work. Is this a result of training using this repository or your re-implementation?

**Comment by fengpany** [comment]
> Hey, thank you for your interest in this work. Is this a result of training using this repository or your re-implementation?

Yes,this is the result of replicating this repository code.I can't reproduce to the accuracy of the paper.

**Comment by fengpany** [comment]
Do you have a weight file that achieves the accuracy of the paper, can you send it to me?I want to reproduce to the accuracy of the paper to facilitate my subsequent research.
